tests/src/RunBenchmarks/benchmarks.xml

```diff
@@ -0,0 +1,669 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<benchmark-system>
+    <benchmark-root-directory>\tf\CodeGen\src\QA\CLR\CQPerfSuite\benchmarks</benchmark-root-directory>
+    <benchmark-suite>
+        <name>Roslyn</name>
+            <benchmark>
+                <name>Roslyn</name>
+                <directory>.\RoslynSep13\SourcesSnapShot</directory>
+                <executable>..\CompilerBinaries\rcsc.exe</executable>
+                <args>/nologo /optimize+ /noconfig @Repro.rsp</args>
+            </benchmark>
+    </benchmark-suite>
+    <benchmark-suite>
+        <name>System.XML</name>
+            <benchmark>
+                <name>System.XML</name>
+                <directory>.\System.XML\x86ret</directory>
+                <executable>SimpleTest.exe</executable>
+                <args>2000 &gt; out</args>
```

Comment:
I wonder if it's a good idea to include stdout/stderr redirection in the 'args' option. I wonder if that's very portable. Perhaps, instead, there should be a <stdoutredirection> and maybe <stderrredirection> option, that, if set to true, causes the harness to automatically create the proper redirection.